laravel/app/Console/Commands/StatsCleanup.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Services\LoggingService\ILoggingService;
+use Illuminate\Console\Command;
+
+class StatsCleanup extends Command
```

Comment:
This it the stats cleanup task. This can be executed from the commandline via:

```
vagrant ssh
cd /vagrant/laravel
php artisan stats:cleanup
```

Or executed via task scheduler, or PHP code from another location